### PR TITLE
Updates for better behaviour on lambda

### PR DESCRIPTION
**Pull request by RyanMacG**
Lambda expects to pass event and context as arguments to the function,
as a result the function must be able to take two arguments.

We also used this as a chance to add logging

### CODE REVIEW

File: user_deletion.py [hunk 1]
```diff
@@ -1,15 +1,29 @@
...
-    cursor.execute("SHOW DATABASES")
-    return cursor.fetchall()
\ No newline at end of file
+import pymysql
+
+logger = logging.getLogger()
+logger.setLevel(logging.INFO)
+
+try:
+  db = pymysql.connect(
```
Comment (adrianclay) [inline user_deletion.py]: Given this is in global scope now, how many times does this get run now?

File: user_deletion.py [hunk 2]
```diff
@@ -1,15 +1,29 @@
...
+    os.environ.get('DATABASE_PASSWORD'),
+    os.environ.get('DATABASE')
+  )
+except:
+  logger.error("ERROR: Unexpected error: Could not connect to MySql instance.")
+  sys.exit()
+
+logger.info("SUCCESS: Connection to RDS mysql instance succeeded")
+
+def delete_old_users(_event, _context):
```
Comment (adrianclay) [inline user_deletion.py]: These args are unused at the moment?

File: user_deletion.py [hunk 3]
```diff
@@ -1,15 +1,29 @@
...
+  cursor = db.cursor()
+  cursor.execute("SHOW DATABASES")
+  return cursor.fetchall()
+
+logger = logging.getLogger()
+logger.setLevel(logging.INFO)
```
Comment (adrianclay) [inline user_deletion.py]: This variable isn't being used?

File: user_deletion.py (hunk 2 above)
Comment (RyanMacG) [reply user_deletion.py]: Yes

**Pull request merged**